lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidator.test.js

```diff
@@ -0,0 +1,421 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../../tests/utils/run-serverless');
+const fixtures = require('../../../../../../../../tests/fixtures');
+
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe('#compileRequestValidators()', () => {
+  after(fixtures.cleanup);
+
+  describe(' - provider configuration -', () => {
```

Comment:
Let's stick to convention of a project, so no ` - ` prefixes and postfixes.

Also let's maybe name it  `reusable schemas`, note `provider configuration`, suggests it's a configuration of provider.